docker-compose.yml

```
@@ -15,6 +15,7 @@ services:
       - external_network
     volumes:
       - ./files:/misskey/files
+      - ./.config:/misskey/.config
```

```suggestion
      - ./.config:/misskey/.config:ro
```

roにする例を多く見るので付けたほうがいいかも

---

サジェストコミットしたけどこれ思いっきり人のdevelopブランチだった。(プルリク時に許可してるからまあいいんだろうけど)
@sousuke0422

---

`:no`についてドキュメントみたいなのある？

---

https://docs.docker.com/storage/volumes/#use-a-read-only-volume
かな

---

別にdevelopだけど、使ってないから問題ないね